subsys/net/l2/openthread/Kconfig

```diff
@@ -261,6 +261,18 @@ config OPENTHREAD_NCP_RADIO
 	help
 	  Enable NCP in OpenThread stack as radio-only
 
+config OPENTHREAD_NCP_VENDOR_HOOK
+	bool "Enable NCP vendor hook"
+        depends on OPENTHREAD_NCP
+        help
+	  Enable vendor specific code for NCP
+
+config OPENTHREAD_NCP_VENDOR_HOOK_SOURCE
+	string "NCP vendor hook source file"
```

Comment:
I think this entry should be depending on OPENTHREAD_NCP_VENDOR_HOOK, i. e. 
`depends on OPENTHREAD_NCP_VENDOR_HOOK`

Reply:
I have reduced it to only one option

Reply:
Ok, so now this config will always be enabled when we enable NCP. Will it work, if we don't want to use vendor hook and we leave this option as an emtpy string?

Reply:
Yes it works with an empty string.
With an empty string the if from https://github.com/zephyrproject-rtos/openthread/pull/13/commits/35d3bdfa71f850de764dbdb3ec01252d4cb853c8 doesn't set the option OT_NCP_VENDOR_HOOK_SOURCE so openthread is not using or expecting a ncp vendor hook.

Reply:
OK, I see, thanks.